aegis_control/launch/start_drivers.launch.py

```diff
@@ -87,6 +100,7 @@ def launch_setup(context: LaunchContext) -> list[LaunchDescriptionEntity]:
         gripper_driver,
         ft_sensor_driver,
         depthai_cameras_driver,
+        tool_cameras_driver,
```

Comment:
Let's improve the name notation:
```suggestion
        tool_rgb_cameras_driver,
```

Reply:
Done